lib/account.js

```diff
@@ -28,9 +28,15 @@ function parseMessage(message, callback) {
       try {
         json = JSON.parse(body);
       } catch (ex) {
-        return callback(Boom.create(message.statusCode, 'LoginAPI error', body));
+        return callback(new Boom('LoginAPI error', {
```

Comment:
Boom API change here, `Boom.create` does not exist anymore.

I also kept these generic functions as callback-driven vs. switching to promises so as to minimize the number of unnecessary changes to software that we are going to deprecate.